client/my-sites/pages/page-list.jsx

```diff
@@ -203,37 +203,40 @@ var Pages = React.createClass( {
 
 		// pages have loaded, sites have loaded, and we have a site instance or are viewing all-sites
 		if ( pages.length && this.props.sites.initialized ) {
-			if ( ! this.props.search ) {
-				// we're listing in reverse chrono. use the markers.
-				pages = this._insertTimeMarkers( pages );
-			}
-			rows = pages.map( function( page ) {
-				if ( ! ( 'site_ID' in page ) ) {
-					return page;
+			const site = this.props.sites.getSelectedSite();
```

Comment:
@jblz has a PR open to do that separately (#13317) -- I'll make sure that it gets rebased and included in the feature branch (#13250)